examples/fabric/build_your_own_trainer/trainer.py

```diff
@@ -4,7 +4,8 @@
 from typing import Any, cast, Iterable, List, Literal, Optional, Tuple, Union
 
 import torch
-from lightning_utilities.core import apply_to_collection, is_overridden
+from lightning_utilities.core import is_overridden
```

Comment:
This should use the version from `pytorch_lightning`, and then the expected parent instance is not necessary to pass where it is used